README.md

```diff
@@ -16,6 +25,10 @@ If you don't have an account, you can create it here
     - SendGridOptions__SendGridAPIKey
     - SendGridOptions__SenderEmail
     - SendGridOptions__SenderName
+    - GmailOptions__Host
```

Comment:
Update the description:
Enter credentials of your sender in docker-compose.yml.  
If you use the Google SMTP, fill out the following variables:
- GmailOptions__Host
- GmailOptions__Port
- GmailOptions__FromEmail
- GmailOptions__FromPassword  

If you use Send Grid, fill out:
- SendGridOptions__SendGridAPIKey
- SendGridOptions__SenderEmail
- SendGridOptions__SenderName